.github/workflows/ci.yml

```diff
@@ -88,6 +88,17 @@ jobs:
           cd backend
           sbt "project mockingbird-api;clean;fixCheck;test;missinglinkCheck;Docker / stage;"
 
+      - name: Set image name
+        run: echo "MOCKINGBIRD_IMAGE=ghcr.io/leviysoft/mockingbird:${{ steps.get-ref.outputs.tag }}" >> $GITHUB_ENV
```

Comment:
Why use `echo`? Let's set up this variable using [github variables](https://docs.github.com/en/actions/learn-github-actions/variables)